conference/program.md

```diff
@@ -231,15 +231,19 @@ redirect_from:
 </tbody>
 </table>
 
-<a class="program-block-anchor" id="keynote-nasma-ahmed,-digital-justice-lab"></a>
-## Keynote
+<a class="program-block-anchor" id="keynote-nasma-ahmed,-navigating-digital-futures"></a>
+## Keynote: Navigating Digital Futures
 {:.program-heading}
 <div class="session-detail program-block">
   <div class="session-detail-content program-block-content">
     <header class="session-header">
       <h3 class="session-title">Nasma Ahmed, Founder and Executive Director of <a href="https://digitaljusticelab.ca/" target="_blank" rel="noopener">Digital Justice Lab</a></h3>
+      <p>Nasma Ahmed is a black muslimah working within the intersections of social justice, technology and policy. She is a technologist and capacity builder based in Toronto. She is the Executive Director of the Digital Justice Lab a national organization focusing on building a more just and equitable digital future in Canada.</p>
```

Comment:
Would it make sense if this came after the two graphics?

Reply:
🤷‍♀️ I tried both ways and felt like this looked cleaner. But ambivalent if you would prefer

Reply:
Ok. I agree on this being cleaner